compiler/rustc_query_impl/src/on_disk_cache.rs

```diff
@@ -505,17 +393,6 @@ impl<'sess> OnDiskCache<'sess> {
         debug_assert!(prev.is_none());
     }
 
-    fn get_raw_def_id(&self, hash: &DefPathHash) -> Option<RawDefId> {
-        self.foreign_def_path_hashes.get(hash).copied()
-    }
-
-    fn try_remap_cnum(&self, tcx: TyCtxt<'_>, stable_crate_id: StableCrateId) -> Option<CrateNum> {
-        let cnum_map = self.cnum_map.get_or_init(|| Self::compute_cnum_map(tcx));
```

Comment:
`cnum_map` and `compute_cnum_map` can be replaced by calls to `TyCtxt::stable_crate_id_to_crate_num`.